routes/errortracker.js

```diff
@@ -0,0 +1,250 @@
+/**
+ * Created by rigwanzojunior on 6/5/17.
+ *objects used
+ *Objects are identical to the objects used in previous errortracker.go
+ * @typedef errorRequestMeta {
+	HTTPReferrer: string,
+	HTTPUserAgent: string
+    }
+  @typedef errorRequest {
+	URL:  string,
+	Method: string,
+	Meta: errorRequestMeta
+  }
+  @typedef errorEvent {
+	Application: string,
+	AppID; string,
+	Environment: string,
+	Version: string,
+	Message: string,
+	Exception: string,
+	Request: *errorRequest,
+	Filename: string,
+	Line: int32,
+	Classname: string,
+	Function: string,
+	Severity: string
+   }
+ * Handle error requests from clients and log them.
+ */
+// AppEngine project ID
+const projectId ='';
```

Comment:
How you will identify your logs on app engine. The id of your app on appengine. It is used for authentication too